pkg/linear-pools/contracts/tetu-linear-pool/TetuLinearPool.sol

```diff
@@ -24,11 +24,8 @@ import "./interfaces/ITetuSmartVault.sol";
 import "./TetuShareValueHelper.sol";
 
 contract TetuLinearPool is LinearPool, Version, TetuShareValueHelper {
-    IERC20 private immutable _mainToken;
     IERC20 private immutable _wrappedToken;
 
-    uint256 private immutable _rateScaleFactor;
```

Comment:
Removed _rateScaleFactor completely. It's not needed because Tetu wrapped token has same decimals as main token (i.e. 18 + wrappedDecimals - mainDecimals will always be 18). It's used in ERC-4626 because we can't be sure about wrapped decimals.

Reply:
This makes sense. I think we should add a comment to `TetuShareValueHelper._getTokenRate()` explaining why the output will always have 18 decimals (i.e., because `mainToken` and `wrappedToken` have the same decimals, and we are doing a `FixedPoint.divDown()` which multiplies by 1e18).